books/RayTracingInOneWeekend.html

```diff
@@ -2510,6 +2447,107 @@
+Indeed, the 70% reflector is closer to middle-gray. The reason for this is that almost all
+computer programs assume that an image is “gamma corrected” before being written into an image
+file. This means that the 0 to 1 values have some transform applied before being stored as a byte.
+Images with data that are written without being transformed are said to be in _linear space_,
+whereas images that are transformed are said to be in _gamma space_. It is likely that the image
+viewer you are using is expecting an image in gamma space, but we are giving it an image in linear
+space. This is the reason why our image appears inaccurately dark.
+
+There are many good reasons for why images should be stored in gamma space, but for our purposes we
+just need to be aware of it. We are going to transform our data into gamma space so that our image
+viewer can more accurately display our image. As a simple approximation, we can use “gamma 2” as our
+transform, which is the power that you use when going from gamma space to linear space. We need to
+go from linear space to gamma space, which means taking the inverse of "gamma 2", so the power
+$1/\mathit{gamma}$, which is just the square-root.
+
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ C++ highlight
+    inline double linear_to_gamma(double linear_component)
+    {
+        return sqrt(linear_component);
+    }
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ C++
+
+    void write_color(std::ostream &out, color pixel_color, int samples_per_pixel) {
+        auto r = pixel_color.x();
+        auto g = pixel_color.y();
+        auto b = pixel_color.z();
+
+        // Divide the color by the number of samples.
+        auto scale = 1.0 / samples_per_pixel;
+        r *= scale;
+        g *= scale;
+        b *= scale;
+
+
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ C++ highlight
+        // Apply the linear to gamma transform.
+        r = linear_to_gamma(r);
+        g = linear_to_gamma(g);
+        b = linear_to_gamma(b);
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ C++
+
+        // Write the translated [0,255] value of each color component.
+        static const interval intensity(0.000, 0.999);
+        out << static_cast<int>(256 * intensity.clamp(r)) << ' '
+            << static_cast<int>(256 * intensity.clamp(g)) << ' '
+            << static_cast<int>(256 * intensity.clamp(b)) << '\n';
+    }
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    [Listing [write-color-gamma]: <kbd>[color.h]</kbd> write_color(), with gamma correction]
+
+
+<div class='together'>
+If we take the gamut now we get a much more consistent ramp from darkness to lightness:
+
+  ![<span class='num'>Image 11:</span> The gamut of our renderer so far
```

Comment:
_gamut_: range or extent. That's not what we mean here. Perhaps change the caption to "A/The gamma-corrected rendering".